osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -129,6 +129,12 @@ protected SongSelect()
                     Right = left_area_padding,
                 },
             });
+            Add(new ActionContainer

```

Comment:
how about calling this `ResetScrollContainer`, so it has meaning. also make the OnHoverAction just a ctor parameter rather than a field. it only serves one purpose and is *required* for this component to have meaning, so ctor works better.